pulumi/aws/observability/otel-objects/otel-collector.yaml.full

```diff
@@ -0,0 +1,65 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: simplest
+  namespace: observability
+spec:
+  config: |
+    extensions:
+      health_check:
+      pprof:
+        endpoint: 0.0.0.0:1777
+      zpages:
+        endpoint: 0.0.0.0:55679
+    
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:9978
+          http:
+            endpoint: 0.0.0.0:9979
+      opencensus:
+      jaeger:
+        protocols:
+          grpc:
+          thrift_binary:
+          thrift_compact:
+          thrift_http:
+      zipkin:
+    
+      # Collect own metrics
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: 'otel-collector'
+              scrape_interval: 120s
+              static_configs:
+                - targets: [ '0.0.0.0:8080']
+              metrics_path: '/z/prometheus'
+    
+    processors:
+      batch:
+    
+    exporters:
+      prometheus:
+        endpoint: "0.0.0.0:8889"
+    
+      logging:
+        logLevel: debug
+    
+      jaeger:
+        endpoint: "0.0.0.0:14250"
+     
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp, opencensus, jaeger, zipkin]
+          processors: [batch]
+          exporters: [logging, jaeger]
+        metrics:
+          receivers: [otlp, opencensus, prometheus]
+          processors: [batch]
+          exporters: [logging]
+    
+      extensions: [health_check, pprof, zpages]
```

Comment:
We may need to check that these extensions are configured correctly or alternately disable them.

Reply:
See note above...